clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/deploy/ops/DeployCloudrunAtomicOperation.java

```diff
@@ -110,6 +120,26 @@ public String deploy(String repositoryPath) {
     return versionName;
   }
 
+  private void populateCloudrunYmlData(List<String> configFiles) throws JsonProcessingException {
+
+    ObjectMapper yamlReader = new ObjectMapper(new YAMLFactory());
```

Comment:
Don't create a new `ObjectMapper` on every invocation. Can we move this to the class level?

Reply:
@mattgogerly ....done..how did i miss it?